src/main/java/org/cyclops/integrateddynamics/core/recipe/type/RecipeSerializerFacadeSqueeze.java

```diff
@@ -0,0 +1,63 @@
+package org.cyclops.integrateddynamics.core.recipe.type;
+
+import org.cyclops.cyclopscore.helper.RecipeSerializerHelpers;
+import org.jetbrains.annotations.Nullable;
+
+import com.google.gson.JsonObject;
+
+import net.minecraft.core.NonNullList;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.minecraft.world.item.crafting.Ingredient;
+import net.minecraft.world.item.crafting.RecipeSerializer;
+import net.minecraftforge.fluids.FluidStack;
+
+public class RecipeSerializerFacadeSqueeze implements RecipeSerializer<RecipeFacadeSqueeze> {
+
+    @Override
+    public RecipeFacadeSqueeze fromJson(ResourceLocation recipeId, JsonObject json) {
+        JsonObject result = GsonHelper.getAsJsonObject(json, "result");
+
+        // Input
+        Ingredient inputIngredient = RecipeSerializerHelpers.getJsonIngredient(json, "item", true);
+
+        // Output
+        NonNullList<RecipeSqueezer.IngredientChance> outputItemStacks = RecipeSerializerSqueezer.getJsonItemStackChances(result, "items");
```

Comment:
Not sure we even need outputs, as it's also always the facade?
I would guess the serializer to require no special JSON params.